libmscore/select.cpp

```diff
@@ -450,6 +450,20 @@ void Selection::appendChord(Chord* chord)
 
 void Selection::updateSelectedElements()
       {
+      if (_state == SelState::RANGE && _plannedTick1 != -1 && _plannedTick2 != -1) {
+            const int staffStart = _staffStart;
+            const int staffEnd = _staffEnd;
+            deselectAll();
+            Segment* s1 = _score->tick2segmentMM(_plannedTick1);
+            Segment* s2 = _score->tick2segmentMM(_plannedTick2, /* first */ true);
+            if (s2 && s2->measure()->isMMRest())
+                  s2 = s2->prev1MM(); // HACK both this and the previous "true"
```

Comment:
It is better to explain the workaround in particular to leave code clear for other developers and for yourself in a year :)

Reply:
I updated comments for both discussed places.